index.d.ts

```diff
@@ -0,0 +1,39 @@
+declare module 'envvar' {
+  /**
+   * Retrieves a given environment variable as a string.
+   *
+   * @throws if the environment variable is not set and a default is not
+   *         specified.
+   * @throws if the value of the environment variable is not a string.
+   */
+  export function string(name: string, defaultValue?: string): string;
+
+  /**
+   * Retrieves a given environment variable as a number.
+   *
+   * @throws if the environment variable is not set and a default is not
+   *         specified.
+   * @throws if the value of the environment variable is not a number.
+   */
+  export function number(name: string, defaultValue?: number): number;
+
+  /**
+   * Retrieves a given environment variable as a boolean.
```

Comment:
I was referring to the fact that Boolean is named after George Boole and is thus conventionally capitalized in prose.